### ISSUE TITLE
Mentions: Fix support for screen readers

### ISSUE THREAD

**Issue by afercia** [draft-js-plugins/draft-js-plugins#805]
In your [online documentation](https://www.draft-js-plugins.com/plugin/mention), under "Supported Environment", I can read that screen readers are supported.

I've tested the mention plugin "Simple Example" online demo with Safari10 + VoiceOver and Firefox + NVDA and neither of those can announce the suggested options while arrowing through the list.

Quickly inspecting the rendered markup, I see there are several things implemented correctly and that's great. However, there are two big things missing. Without them, screen readers can't really work.

1
the `aria-activedescendant` attribute on the element that has role "combobox" gets a new value while highlighting the options but this value _must_ be the same value of the highlighted option ID attribute; as far as I see all the options completely miss an ID

2
`aria-selected="true"` must be set on the highlighted option

![screen shot 2017-07-25 at 16 37 50](https://user-images.githubusercontent.com/1682452/28579079-d8cc71ec-715b-11e7-8be8-384b615cd9fc.png)

As a result, Safari10 + VoiceOver don't announce anything while highlighting the options. Firefox + NVDA announce always the first option. Haven't tested with IE11 + JAWS. Worth noting these 3 browser / screen readers combinations are the best options, since each screen reader works "best" with one browser. (Note: JAWS works well with Firefox too).

![screenshot 3](https://user-images.githubusercontent.com/1682452/28579220-442ebdbe-715c-11e7-98ec-d1d5ca17eeed.png)

As a reference, I'd like to point to the ARIA Authoring Practices but unfortunately the section dedicated to the combobox widget is currently seeing some rework:
https://www.w3.org/TR/wai-aria-practices/#combobox
they point to the related Github issue:
https://github.com/w3c/aria-practices/issues/31
where you can find links to the previous draft:
https://rawgit.com/w3c/aria-practices/master/aria-practices-DeletedSectionsArchive.html#combobox
https://rawgit.com/w3c/aria-practices/master/aria-practices-DeletedSectionsArchive.html#autocomplete

Also: ARIA 1.1: `aria-activedescendant`
https://www.w3.org/TR/wai-aria-1.1/#aria-activedescendant

> For example, in a combobox, focus may remain on the textbox while the value of aria-activedescendant on the textbox element refers to a descendant of a popup listbox that is controlled by the textbox.

> Value: ID reference

**Comment by nikgraf** [comment]
That's awesome feedback! Thanks you very much for looking into it

**Comment by juliankrispel** [comment]
@afercia really great feedback! Will be looking into fixing this shortly

**Comment by afercia** [comment]
The issue 2) about aria-selected is being addressed in https://github.com/draft-js-plugins/draft-js-plugins/pull/1075

Will split the issue 1) about aria-activedescendant in a separate issue and close this one.
